gcsfs/core.py

```diff
@@ -957,17 +979,29 @@ def getxattr(self, path, attr):
         meta = self.info(path).get('metadata', {})
         return meta[attr]
 
-    def setxattrs(self, path, **kwargs):
-        """ Set user-defined metadata attributes
+    def setxattrs(self, path, content_type = None, content_encoding=None, **kwargs):
+        """ Set/delete/add writable metadata attributes
 
         Parameters
         ---------
-        kw_args : key-value pairs like field="value", where the values must be
-            strings. Replaces existing metadata.
+        content_type: str
+            Sets the Content-Type. Note: gcsfs defaults to 'application/octet-stream' on uploads
```

Comment:
"If not specified, defaults to 'application/octet-stream'"

Reply:
Not sure as the method/request itself would not defaults or change the content_type. It would leave it as initially set, that is `application/octet-stream` unless https://github.com/dask/gcsfs/pull/124 is implemented

Suggested: "Changes the Content-Type"

Reply:
If not None, set the content-type to this value